app/models/reporting/data_quality_reports/enrollment.rb

```diff
@@ -84,10 +84,10 @@ module Reporting::DataQualityReports
   class Enrollment < ReportingBase
     include ArelHelper
 
-    COMPLETE = [1, 2]
-    REFUSED = [8, 9]
-    YES_NO = [0,1]
-    VALID_GENDERS = [0,1,2,3,4]
+    COMPLETE = [1, 2].freeze
+    REFUSED = [8, 9].freeze
+    YES_NO = [0, 1].freeze
+    VALID_GENDERS = [0, 1, 2, 3, 4].freeze
```

Comment:
This doesn't seem right anymore... But I think it isn't used, so it can just be deleted.